### DISCUSSION TITLE
Gap/Gutter utility classes for d-flex?

### DISCUSSION THREAD

**Discussion by sweeperq** [twbs/bootstrap#39427] [Q&A] [answered]
When I try to set a gap on a parent flex item it doesn't work. Example: `<div class="d-md-flex g-3">`. When I manually use the style attribute like this it works: `<div class="d-md-flex" style="gap: .5rem">`. Is there something else I'm supposed to be using for gaps/gutters with flex?

**Comment by julien-deramond** [comment, marked as answer]
In this case, you're supposed to use `.gap-3` instead of `.g-3` (see [Utilities > Spacing > Gap](https://getbootstrap.com/docs/5.3/utilities/spacing/#gap)).

**Reply by sweeperq** [reply]
Thanks, exactly what I was looking for.
